stable/wordpress/templates/deployment.yaml

```diff
@@ -28,7 +28,11 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
         env:
         - name: ALLOW_EMPTY_PASSWORD
-          value: {{ .Values.allowEmptyPassword | quote }}
+        {{- if .Values.allowEmptyPassword }}
```

Comment:
won't "no" be evaluated as true as well? https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md

Reply:
@davidkarlsen you're right, what if we change the Helm value to actually be a boolean (works well with `--set`), and then make sure the template always renders "yes" or "no" strings depending on the value of the boolean?